pkg/ruler/base/ruler.go

```diff
@@ -514,10 +514,10 @@ func (r *Ruler) listRules(ctx context.Context) (result map[string]rulespb.RuleGr
 		return
 	}
 
-	for userID := range result {
-		if !r.allowedTenants.IsAllowed(userID) {
-			level.Debug(r.logger).Log("msg", "ignoring rule groups for user, not allowed", "user", userID)
-			delete(result, userID)
+	for orgID := range result {
+		if !r.allowedTenants.IsAllowed(orgID) {
+			level.Debug(r.logger).Log("msg", "ignoring rule groups for user, not allowed", "user", orgID, "org_id", orgID)
```

Comment:
@owen-d and @dannykopping since you are on the original ticket. Should I drop `"user"`. I wonder whether this is going to break current queries.